app.js

```diff
@@ -1,21 +1,31 @@
 const { getTopics } = require("./controllers/topics.controllers")
 const { getEndpoints } = require("./controllers/api.controllers")
+const { getArticleById } = require("./controllers/articles.controllers")
 
 const express = require("express")
 
 const app = express()
 
-app.use(express.json())
-
 app.get("/api/topics", getTopics)
 
 app.get("/api", getEndpoints)
 
+app.get("/api/articles/:article_id", getArticleById)
+
+app.all('*', (req, res) => {
```

Comment:
Lovely! Potentially the test above is required then if you have only just added this but it might be better in it's own describe block as it works for any misspelled endpoint / random other endpoint that was requested and is not specific to the /api/articles/:article_id endpoint.